libr/core/vmenus.c

```diff
@@ -153,7 +153,7 @@ R_API bool r_core_visual_esil(RCore *core) {
 		// bool use_color = core->print->flags & R_PRINT_FLAGS_COLOR;
 		(void) r_asm_disassemble (core->assembler, &asmop, buf, sizeof (ut64));
 		analop.type = -1;
-		(void)r_anal_op (core->anal, &analop, core->offset, buf, sizeof (ut64), R_ANAL_OP_MASK_ESIL);
+		(void)r_anal_op (core->anal, &analop, core->offset, buf, sizeof (ut64), R_ANAL_OP_MASK_ALL);
```

Comment:
Why do u need R_ANAL_OP_MASK_ALL , where only MASK_ESIL is needed ?

Reply:
To be fair, I have absolutely no idea why that line changed.
I didn't even acknowledge that code, I only wrote mine and fixed spaces around.

Reply:
Ah np :) , can u try to rebase this on top of master, may solve the problem